File: diane-kpis-service/diane-kpis-common/src/main/java/io/myplant/model/DeviceStatusV2.java
package io.myplant.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class DeviceStatusV2 {
    private EngineAction ActionActual = EngineAction.UNDEFINED;

    private long ActionFrom;
    private long ActionTo;

    private long TriggerDate;
    private long TriggerMSGNo;
    private String TriggerText;
    private String TriggerResponsibility;
    private long TriggerCount;

    @JsonProperty("dss")
    private DemandSelectorSwitchStates DemandSelectorSwitch;
    @JsonProperty("sss")
    private ServiceSelectorSwitchStates ServiceSelectorSwitch;
    @JsonProperty("avss")
    private AvailableStates AV_MAN_Activated_Status;

    public String toString() {
        return "DeviceStatusV2(ActionActual=" + this.getActionActual() + ", ActionFrom=" + new Date(this.getActionFrom()) + ", actionTo=" + new Date(this.getActionTo())
                + ", TriggerDate=" + new Date(this.getTriggerDate()) + ", TriggerMSGNo=" + this.getTriggerMSGNo()
                + ", DemandSelectorSwitch=" + this.getDemandSelectorSwitch() + ", ServiceSelectorSwitch=" + this.getServiceSelectorSwitch()
                + ", AV_MAN_Activated_Status=" + this.getAV_MAN_Activated_Status() + ")";
    }
}

File: service-net/commons.net/commons.net/Performance/DeviceStatusV2.cs
using GE.Myplant.Integration.NetCore.Utils;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System;

namespace GE.Myplant.Integration.NetCore.Performance
{
    public class DeviceStatusV2 
    {
        public EngineAction ActionActual { get; set; } = EngineAction.Undefinded;

        [JsonConverter(typeof(MillisecondEpochConverter))]
        public DateTime? ActionFrom;
        [JsonConverter(typeof(MillisecondEpochConverter))]
        public DateTime? ActionTo;

        [JsonConverter(typeof(MillisecondEpochConverter))]
        public DateTime? TriggerDate;
        public long TriggerMSGNo;
        public String TriggerText;
        public String TriggerResponsibility;
        public long TriggerCount;

        [JsonProperty("dss")]
        public DemandSelectorSwitchStates DemandSelectorSwitch;
        [JsonProperty("sss")]
        public ServiceSelectorSwitchStates ServiceSelectorSwitch;
        [JsonProperty("avss")]
        public AvailableStates AV_MAN_Activated_Status;

        [JsonConverter(typeof(MillisecondEpochConverter))]
        public DateTime? CalcDate;
    }

}
